Framework/PDFExporter/PDFExporter/Categories/UIKit/UITextView+PDFExporterDrawing.m

```diff
@@ -0,0 +1,16 @@
+//
+//  UITextView+PDFExporterDrawing.m
+//  PDFExporter
+//
+//  Copyright © 2016 3Pillar Global. All rights reserved.
+//
+
+#import "UITextView+PDFExporterDrawing.h"
+
+@implementation UITextView (PDFExporterDrawing)
+
+- (BOOL)canDrawSubview:(UIView *)subview intersection:(CGRect)intersection {
```

Comment:
Would be nice if cases like this one were documented, it's hard to understand what's happening

Reply:
We have written documentation for source code.